libs/components/tsconfig.lib.json

```diff
@@ -1,14 +1,11 @@
 {
     "extends": "./tsconfig.json",
-    "files": ["../../cypress/types.d.ts"],
+    "files": ["../../cypress/types.d.ts", "./src/progress-bar/newFile.ts"],
     "compilerOptions": {
         "sourceMap": true,
         "inlineSources": true,
         "declaration": true,
         "types": ["cypress", "cypress-axe"]
     },
-    "exclude": [
-        "jest.config.ts",
-        "**/*.spec.ts"
-    ]
+    "exclude": ["jest.config.ts", "**/*.spec.ts"]
```

Comment:
Als het kan, dit niet op een andere manier formatteren behalve als er een wijziging nodig is.

Reply:
dat gebeurde automatisch op mijn IDE, ik weet het niet waarom.

Reply:
`"./src/progress-bar/newFile.ts"],` is verwijderd maar er is nog altijd een wijziging in de `exclude` kan je die reverten?

Reply:
gedaan